tic_tac_toe/game.py

```diff
@@ -33,10 +38,13 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+    for position in board:
+        if "-" in position:
+            return False
+    return True
 
 
-def _is_winning_combination(board, combination, player):
+def _is_winning_combination(combination, player):
```

Comment:
I'm not sure it's necessarily a good idea to be changing your interface here. We don't currently have unit tests calling this exact function but if we did making a change like this would be breaking those tests. Your code could have been written to achieve the same results without altering the function definition.